builds/fabric8-che/plugins/che-plugin-bayesian-lang-server/src/main/java/org/eclipse/che/plugin/languageserver/bayesian/server/launcher/BayesianLanguageServerLauncher.java

```diff
@@ -87,8 +87,10 @@ private static LanguageServerDescription createServerDescription() {
                                                                               null),//
                                                            new DocumentFilter(JsonModule.LANGUAGE_ID,//
                                                                               "package\\.json",//
-                                                                              null)//
-                                             // TODO pom.xml
+                                                                              null),
+                                                           new DocumentFilter("pom",
```

Comment:
It probably makes sense to extract the launguage id "pom" to a constant when we start using it outside of the maven module.